### ISSUE TITLE
Questions regarding the paper

### ISSUE THREAD

**Issue by RishabhMaheshwary** [JHL-HUST/FGPM#1]
1. In adversarial training, Are you augmenting the adversarial examples in the original training set, or are you just using the logits of the generated adversarial examples and use it in the loss function and train the model on the original training set?
2. Are you using the Mean Squared Error Loss to calculate the regularization term?
